tests/unittest_brain.py

```diff
@@ -1398,12 +1420,12 @@ class Derived1(MutableSet[T]):
         """
         )
         inferred = next(node.infer())
-        check_metaclass_is_abc(inferred)
         assertEqualMro(
             inferred,
             [
                 "Derived1",
                 "MutableSet",
+                "MutableSet",
```

Comment:
Could this be a set with unique value ?

Reply:
It basically is. If you resolve the ClassDef with `qname`, you would get
```py
typing.MutableSet
collections.abc.MutableSet
```
`assertEqualMro` only checks the `name` of the `ClassDef`. That's why they appear the same.

Reply:
Ho, ok I get it now. We could do the test on `qname` so the test is more explicit then ?

Reply:
I've updated the tests to use `qname()`